### DISCUSSION TITLE
About hammerdbcl Scripting Behavior

### DISCUSSION THREAD

**Discussion by mamiyaakira** [TPC-Council/HammerDB#347] [Q&A] [answered]
I am reading a script to run hammerdbcli.
The vurun does not terminate.
vucomplete continues to run with false. How to deal with it
If you have any, please let me know.

$ ./hammerdbcli < input_tpcc.tcl

- input_tpch.tcl

dbset db pg
dbset bm TPROC-H
diset connection  pg_host localhost
diset connection pg_port 5432
diset tpch pg_tpch_superuser ace14327nx
diset tpch pg_tpch_user ace14327nx
diset tpch pg_tpch_dbase tpch
diset tpch pg_scale_fact 1
diset tpch pg_num_tpch_threads 4
loadscript
vuset vu 1
vuset logtotemp 1
vuset unique 1
vucreate
vustatus
vurun
waittocomplete

**Comment by mamiyaakira** [comment]
hammerdb runs on centos 8 on 4.4.

**Comment by sm-shaw** [comment, marked as answer]
Start with the [HammerDB CLI 101 ](https://www.hammerdb.com/blog/uncategorized/hammerdb-cli-101/)
Documentation is [here](https://www.hammerdb.com/docs/ch09s08.html) 
The HammerDB CLI will accept the argument auto to run a specified script automatically.

`hammerdbcli.bat auto autorunbuild.tcl`

**Reply by mamiyaakira** [reply]
It worked. Thank you very much.
